plotly/graph_objs/graph_objs.py

```diff
@@ -836,6 +836,20 @@ def __init__(self, *args, **kwargs):
                                                   _parent_key='data')
     figure_class.__init__ = __init__
 
+    def __setitem__(self, key, value, **kwargs):
+        if key == 'frames':
```

Comment:
Let's add a comment about this one. Maybe tie it to a new issue to better-integrate `frames` into `Figure`? As always, pls put the issue numer (or link) in the `TODO` comment.